pages/common/ansible.md

```diff
@@ -1,10 +1,7 @@
 # ansible
 
-> Ansible installation can be found here.
-> [docs.ansible]<https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>.
-
 > The following commands are examples of required steps to ensure ansible is installed properly.
-> Ansible may require a separate user account with sudo privs on each node to run system commands.
+> docs.ansible: <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>.
```

Comment:
```suggestion
> More information: <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>.
```

This should fix the issue.

Reply:
I realize this cheatsheet modification isn't meant for tldr.  I am creating a repo designed for opensource cheatsheets that are not 1liners as it is formatted in tldr.  Closing Pull request.